commerce-add-ons/packages/product-feeds/README.md

```diff
@@ -0,0 +1,16 @@
+---
+description: >-
+  This is the documentation for the Product Feed package for Umbraco Commerce.
+---
+
+## Introduction
+After building an e-commerce website, you might want to share your products with an external service like Google Merchant Center to reach more viewers.
```

Comment:
I like the suggestion made by Bolette as an intro, also because it is written that is open-source which the 2 other packages mention as well:
```suggestion
After building up your commerce website, you can expand your product reach and show your products to a larger audience with product feeds. Umbraco Commerce Product Feed is a free and open-sourced add-on for Umbraco Commerce that helps you generate the XML files that are used in services like Google Merchant Center.
```